packages/wrangler/pages/functions/template-worker.ts

```diff
@@ -1,14 +1,12 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
-
 import { match } from "path-to-regexp";
 import type { HTTPMethod } from "./routes";
 
 /* TODO: Grab these from @cloudflare/workers-types instead */
-type Params<P extends string = any> = Record<P, string | string[]>;
+type Params<P extends string = string> = Record<P, string | string[]>;
 
 type EventContext<Env, P extends string, Data> = {
   request: Request;
-  waitUntil: (promise: Promise<any>) => void;
+  waitUntil: (promise: Promise<unknown>) => void;
```

Comment:
I don't think `waitUntil` can return anything, so this should probably be `void`
```suggestion
  waitUntil: (promise: Promise<void>) => void;
```

Reply:
It doesn't _do_ anything with the return, but I can still return something if I choose.

Reply:
`waitUntil(Promise.resolve(5))` is totally valid.